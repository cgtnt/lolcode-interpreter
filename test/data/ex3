O RLY?
  YA RLY
    <code block>
  NO WAI
    <code block>
OIC
