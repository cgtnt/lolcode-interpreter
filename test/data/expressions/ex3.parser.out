BOTH SAEM SUM OF PRODUKT OF 2 3 QUOSHUNT OF 12 4 10